website/pages/docs/plugins/sources/overview.mdx

```diff
@@ -2,7 +2,7 @@
 title: Sources
 ---
 
-import { getLatestVersion } from "../../../utils/versions";
+import { getLatestVersion } from "../../../../utils/versions";
```

Comment:
We should update the util in `versions` to reference the new location of versions under website, here:
https://github.com/cloudquery/cloudquery/blob/2ceb3214cd504a134f9087e8da5a167e9b11e8e5/website/utils/versions.ts#L4